src/Elders.Cronus.Projections.Cassandra/Infrastructure/TableRetention/RetainOnlyNewestProjectionRevisions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Cassandra;
+using Microsoft.Extensions.Options;
+
+namespace Elders.Cronus.Projections.Cassandra.Infrastructure
+{
+    public class RetainOnlyNewestProjectionRevisions : IProjectionTableRetentionStrategy
```

Comment:
I think we should rename the class. It says that only the newest revision will be retained but it is possible to configure how many version could be retained.